css-flexbox/align-items.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      /* A new a new css rule set selecting all elements in the 'card-deck' id ... Opening curly bracket*/
+      #card-deck {
+        display: flex;
+        justify-content: center;
+        min-height: 100vh;
+        align-items: center;
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'card-wrapper' id ... Opening curly bracket*/
+      .card-wrapper {
+        flex-basis: calc(100% / 3);
+        justify-content: center;
```

Comment:
`justify-content` is a parent property which means it gets applied on elements that also have `display: flex`. In this case, it should be removed

Reply:
What a silly mistake thank you!